test/extended/util/image/image.go

```diff
@@ -61,6 +61,8 @@ func init() {
 		"k8s.gcr.io/sig-storage/snapshot-controller:v3.0.2":       -1,
 		"k8s.gcr.io/sig-storage/snapshot-controller:v4.0.0":       -1,
 
+		"k8s.gcr.io/e2e-test-images/busybox:1.29-1": -1,
```

Comment:
I think this would be easier to maintain if it were a few lines down with busybox:1.29-2

Reply:
nah, I prefer to have it next to sig-storage elements, b/c they are handled differently than that other busybox image, it's complicated :sweat: